### PR TITLE
CLI Improvements -> ARTEMIS-4384 and ARTEMIS-4384

### PR DESCRIPTION
**Pull request by clebertsuconic**
this new cli command will verify if the topology is matchin on every cluster member.

### CODE REVIEW

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java [hunk 1]
```diff
@@ -85,6 +87,9 @@ public enum OPERATION {
    @Option(names = "--maxColumnSize", description = "The max width of data column. Set to -1 for no limit. Default is 25.")
    private int maxColumnSize = DEFAULT_MAX_COLUMN_SIZE;
 
+   @Option(names = "--clustered", description = "Print statics for all the nodes on the topology")
```
[the remark this answers is not in the document]
Comment (clebertsuconic) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java:90]: Not sure what you're asking.  Rename statistics as stats or renaming clustered as stats.

[the remark this answers is not in the document]
Comment (gemmellr) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java:90]: I'm suggesting the added "statics" looks wrong and asking for it to be corrected.

[the remark this answers is not in the document]
Comment (clebertsuconic) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java:90]: Ahhhh.  I couldn't even see my typo now. Thanks.

File: artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConnectionAbstract.java [hunk 2]
```diff
@@ -233,6 +235,9 @@ void saveConnectionInfo(String brokerURL, String user, String password) {
       if (Shell.inShell() && CONNECTION_INFORMATION.get() == null) {
          CONNECTION_INFORMATION.set(new ConnectionInformation(brokerURL, user, password));
          System.out.println("CLI connected to broker " + brokerURL + ", user:" + user);
+         this.brokerURL = brokerURL;
+         this.user = user;
+         this.password = password;
```
[the remark this answers is not in the document]
Comment (clebertsuconic) [reply artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/ConnectionAbstract.java:240]: There was a scenario where recoverConnect was called. the tryConnect was called, a failure happened, the user / password/ URI was asked, and then StatQueue didn't have the proper user. 

It's either that, or calling recover at the end (which would be worst).

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlTest.java [hunk 3]
```diff
@@ -257,6 +257,13 @@ public void testSecurityCacheSizes() throws Exception {
       Wait.assertEquals(usingCore() ? 8 : 1, () -> serverControl.getAuthorizationCacheSize());
    }
 
+   @Test
+   public void testCurrentTime() throws Exception {
+      long time = System.currentTimeMillis();
+      ActiveMQServerControl serverControl = createManagementControl();
+      Assert.assertTrue(serverControl.getCurrentTimeMillis() >= time);
```
[the remark this answers is not in the document]
Comment (clebertsuconic) [reply tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlTest.java:264]: I wouldn't check for a delta. As the operation should return the current time. The second call must be >=.

**Comment by clebertsuconic** [comment]
@gemmellr I addressed all the changes.. thanks for your review

**Pull request merged**